spec/system/visit_user_journey_view_certificate_page_spec.rb

```diff
@@ -35,14 +31,20 @@
 
   context 'shows existing msa' do
     it 'encryption certificate information and navigates to next page if not being deployed' do
-      travel_to Time.now + 11.minutes
+      expect(CERT_STATUS_UPDATER).to receive(:update_hub_usage_status_for_cert).and_call_original.with(any_args).at_least(:once)
+      expect(SCHEDULER).to receive(:mode).and_call_original.at_least(:once)
+      create(:replace_encryption_certificate_event, component: msa_encryption_certificate.component, encryption_certificate_id: msa_encryption_certificate.id)
+      wait_until { Certificate.find_by_id(msa_encryption_certificate).in_use_at }
```

Comment:
I would prefer to use the `travel_to` since we know how long to wait (i.e. the cache has a known value)

Reply:
I made changes requested